src/kernel/kmain.zig

```diff
@@ -14,18 +14,18 @@ const panic_root = @import("panic.zig");
 pub fn panic(msg: []const u8, error_return_trace: ?*builtin.StackTrace) noreturn {
     @setCold(true);
     arch.disableInterrupts();
+    asm volatile ("xchgw %%bx, %%bx");
```

Comment:
What does this do?

Reply:
This is a breakpoint for Bochs as i was getting panics in bochs and not in qemu, Will remove